ldap/src/main/java/org/springframework/security/ldap/server/UnboundIdContainer.java

```diff
@@ -54,9 +54,31 @@
 
 	public UnboundIdContainer(String defaultPartitionSuffix, String ldif) {
 		this.defaultPartitionSuffix = defaultPartitionSuffix;
+		setLdif(ldif);
+	}
+
+	public String getLdif() {
```

Comment:
Is it necessary to add a setter and getter for the LDIF file?

Reply:
I think we can do some of the validation on setter level